.github/workflows/argilla.yml

```
@@ -21,7 +21,7 @@ jobs:
   build:
     services:
       argilla-server:
-        image: argilladev/argilla-hf-spaces:develop
+        image: argilladev/argilla-hf-spaces:pr-5572
```

TODO: Apply this change before merge the PR. 

```suggestion
        image: argilladev/argilla-hf-spaces:develop
```